docs/release-notes/release-notes-0.18.2.md

```diff
@@ -86,6 +86,13 @@
 
 ## Testing
 ## Database
+
+* [Fixed](https://github.com/lightningnetwork/lnd/pull/8854) pagination issues 
+  in SQL invoicedb queries.
+
+* [Migrate](https://github.com/lightningnetwork/lnd/pull/8855) incorrectly
```

Comment:
It would be good to mention in the release notes as well that after the migration the default expiry of the invoices will be 24 Hours

Reply:
Done, ptal.